src/constants/PublicWalletXpubs.ts

```diff
@@ -0,0 +1,17 @@
+// FIXME: @0Xf11nix - Do not commit to Prod, replace with actual creds
```

Comment:
heyo - please replace with the demo wallet xpubs in the comment here

https://github.com/shapeshift/hdwallet/pull/519#issuecomment-1111496932

Reply:
Addressed in https://github.com/shapeshift/web/pull/1585/commits/94d0ea775f946cb294188c3a45c2e6a86109e075